src/components/VipCore/vipCore.client.js

```diff
@@ -155,6 +157,9 @@ function parseFindLibraryResponse(response, checkLibraries) {
       if (agency.geolocation) {
         item.distance = getAgencyField(agency.geolocation, 'distanceInMeter');
       }
+      if (branchId !== item.loginAgencyId) {
```

Comment:
Bare for at skrive noget....

Kunne denne if ikke komme ned hvor du skriver til `console.log` ?
Det ser ud til at du kun bruger useLoginAgency det ene sted.

Reply:
Den kan ikke flyttes. Det er en opsamling af de agencies, som logger ind via et andet agency. Listen logges når loopet er færdigt.